sdk/websites/Azure.ResourceManager.AppService/api/Azure.ResourceManager.AppService.netstandard2.0.cs

```diff
@@ -7488,7 +7451,7 @@ public enum TriggeredWebJobStatus
     public partial class Twitter
```

Comment:
The type name also needs a change - it is a single word
But we could hold on this for a better candidate of the name